.github/analyst_guide_data_validation.md

```diff
@@ -0,0 +1,171 @@
+- `Date` variable is showing up as a variable type that is neither _character_ nor _Date_ type
+
+## Data Validation Report
+
+The easiest way to perform data validation with the **wpa** package is to run the data validation report:
+
+```R
+ # `spq_df` is your Standard Person Query data
+validation_report(spq_df)
+```
+This generates an HTML report in your working directory which contains a series of checks against your data, including:
+
+- Workplace Analytics Settings
+- Organizational Data Quality
+- M365 Data Quality
+
+You can find a demo output of the validation report [here](https://microsoft.github.io/wpa/report-demo/validation-report-demo.html).  Note that `validation_report()` only runs with a Standard Person Query, but you can supply an optional meeting query to include checks against meeting subject lines. To do so, you should run: 
+
+```R
+# Assuming:
+# `spq_df` is your Standard Person Query data
+# `mt_df` is your Standard Meeting Query data
+validation_report(spq_df, 
+                  meeting_data = mt_df)
+```
+
+The data validation report provides you with a series of recommendations on whether you should adjust certain settings or consider certain assumptions before proceeding with your analysis. After you've made the relevant adjustments, you can run the 'cleaned' dataset through `validation_report()` again to make sure that the potential issues have been caught out. 
+
+Note that `validation_report()` only provides recommendations based on common scenarios observed with Workplace Analytics data. When deciding whether to make an adjustment, you should consider other factors such as quality of organizational data, context, and other known collaboration norms within the organization. 
+
+## Individual functions
+
+The **wpa** package provides additional data validation functions to be used prior to embarking on a new analysis. These functions make up the majority of the automated checks of `validation_report()`, where you can run them individually to extract more detailed information (for instance, the report may identify certain employees as _non-knowledge workers_, but does the distribution of these workers with respect to organization make sense? ). The key data validation functions are described below.
+
+### Organizational attributes
+
+ `check_query()` performs a check on a query (data frame) and returns a diagnostic message about the data query to the R console, with information such as date range, number of employees, HR attributes identified, etc.
+
+```R
+check_query(sq_data)
+```
+
+`hrvar_count()` enables you to create a count of the distinct people by the specified HR attribute:
+
+```R
+hrvar_count(sq_data, hrvar = "LevelDesignation")
+```
+
+To run a blanket analysis for all the organizational attributes in the dataset, you can run `hrvar_count_all()` instead. 
+
+Also check out: 
+
+- `identify_privacythreshold()`
+- `track_HR_change()`
+- `identify_tenure()`
+
```

Comment:
- `identify_privacythreshold(data_clean, return = "table")`
- `track_HR_change(data_clean)`
- `identify_tenure(data_clean)`

Reply:
Think it might be easier to keep this simple here, so users can refer to the original function documentation